sorting/controller.go

```diff
@@ -0,0 +1,30 @@
+package sorting
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/qor/qor/admin"
+)
+
+func updatePosition(context *admin.Context) {
+	if result, err := context.FindOne(); err == nil {
+		if position, ok := result.(positionInterface); ok {
+			if pos, err := strconv.Atoi(context.Request.URL.Query().Get("pos")); err == nil {
+				if MoveTo(context.GetDB(), position, pos) == nil {
+					context.Writer.Write([]byte("OK"))
+					return
+				}
+			}
+		}
+	}
+	context.Writer.Write([]byte("Error"))
+}
+
+func (s *Sorting) InjectQorAdmin(res *admin.Resource) {
```

Comment:
this is actually `InjectINTOQorAdmin` right? i would rename it - the name suggests that it injects QOR admin into something - not something into QOR admin.